config.example.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I can remove these example files if you want, but they are *super* helpful if anyone else ever wants to contribute to this project (so are migrations or at least some sort of sql script btw 🥺 ).